query/src/test/resources/nrt-performance-writer.xml

```diff
@@ -24,7 +24,7 @@
                <indexed-entity>org.infinispan.query.test.Person</indexed-entity>
             </indexed-entities>
             <!-- Enabled fastest writer: NRT backend -->
-            <property name="index_defaults.io.commit_interval">10000</property>
+            <property name="io.commit_interval">10000</property>
```

Comment:
FWIW, NRT is enabled by default. This doesn't enable it, it just changes the commit interval from 1,000ms to 10,000ms.

Reply:
yes, I am aware. This file is reminiscent of a very old test that needs to be revisit after  due to #8815 too